src/config/mod.rs

```diff
@@ -67,13 +105,127 @@ impl General {
     }
 }
 
+pub fn enabled() -> bool {
```

Comment:
Can't find any consumers of these three -- leftover from earlier?

Reply:
These are used for the serde defaults which need to be functions.